charts/virtaitech-orion-vgpu/v1.0.0/templates/controllernodeportservice.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: orion-controller-nodeport
+spec:
+  type: NodePort
+  selector:
+    app: orion
+    name: {{ .Values.controller.name }}
+  ports:
+  - protocol: TCP
+    port: {{ .Values.controller.port | atoi | add 2 }}
+    targetPort: {{ .Values.controller.portName2 }}
+    nodePort: {{ .Values.controller.nodeport | add 2 }}
```

Comment:
This value is not referenced elsewhere, so why add 2？

Reply:
Previously I actually opened three ports. I removed the other two ports later while forgot to change the port number of the remaining one. I've changed the port number according to this comment.